src/components/content/catalog/services/UpdateService.tsx

```diff
@@ -0,0 +1,229 @@
+                            );
+                            setIsYamlSyntaxValid(true);
+                        } catch (e: any) {
+                            console.log(e);
+                            setYamlSyntaxValidationResult(<Alert type={'error'} showIcon={true} message={e.message} />);
+                            setIsYamlSyntaxValid(false);
+                        }
+                    }
+                };
+            }
+            setYamlSyntaxValidationStatus('completed');
+        }
+    }, [fileList]);
+
+    useEffect(() => {
+        if (registerResult) {
+            if (isRegisterSuccessful) {
+                setRegistrationResultDisplay(
+                    <Alert
+                        type={'success'}
+                        message={`Service ${ocl?.name} Update Successfully`}
+                        closable={true}
+                        onClose={onRemove}
+                        className={'result'}
+                    />
+                );
+            } else {
+                setRegistrationResultDisplay(
+                    <Alert
+                        type={'error'}
+                        closable={true}
+                        showIcon={true}
+                        message={`Service Update Failed`}
+                        description={registerResult}
+                        onClose={onRemove}
+                        className={'result'}
+                        action={
+                            <Button size='small' type='primary' onClick={onRemove} danger={true}>
+                                Try Again
+                            </Button>
+                        }
+                    />
+                );
+            }
+        }
+    }, [registerResult, ocl?.name, isRegisterSuccessful]);
+
+    useEffect(() => {
+        if (isYamlSyntaxValid) {
+            if (isYamlSyntaxValid && ocl !== null && ocl !== undefined) {
+                const oclTableData = DisplayOclData({ ocl: ocl });
+                if (typeof oclTableData === 'string') {
+                    setIsOclDataValid(false);
+                    setOclDataDisplay(
+                        <Alert
+                            type={'error'}
+                            showIcon={true}
+                            message={`OCL data in the uploaded file not Valid. Error while parsing - ${oclTableData}`}
+                        />
+                    );
+                } else {
+                    setIsOclDataValid(true);
+                    setOclDataDisplay(oclTableData);
+                }
+            }
+        }
+    }, [yamlSyntaxValidationResult, ocl, isYamlSyntaxValid]);
+
+    const customRequest = () => {
+        if (fileList) {
+            serviceVendorApi
+                .update(id, ocl as Ocl)
+                .then(() => {
+                    fileList[0].status = 'success';
+                    setRegisterResult('Service Update Successfully');
```

Comment:
result should be -> `Service Updated Successfully`

Reply:
fixed.